manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-role.yaml

```diff
@@ -34,3 +35,10 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
+  - custom.tekton.dev
+  resources:
+  - pipelineloops
```

Comment:
im just wondering do you need to list the pipelineloops? isn't that just the template for customruns?

Reply:
yea, sure! for persistenceagent, let's use minimal permissions. removed it